nltk/corpus/reader/conll.py

```diff
@@ -224,12 +224,12 @@ def _get_tagged_words(self, grid, tagset=None):
             pos_tags = [map_tag(self._tagset, tagset, t) for t in pos_tags]
         return list(zip(self._get_column(grid, self._colmap['words']), pos_tags))
 
-    def _get_iob_words(self, grid, tagset=None):
+    def _get_iob_words(self, grid, tagset=None, columns=['chunk']):
```

Comment:
It's considered a bad practice to use mutable objects as function defaults (it usually leads to nasty bugs that are difficult to fix.

Use either a tuple (a sequence that can't be changed):
```python
def _get_iob_words(self, grid, tagset=None, columns=('chunk', )):
    ...
```
or
```python
def _get_iob_words(self, grid, tagset=None, columns=None):

    if columns is None:
        columns = ['chunk']

    ...
```

The reason mutable defaults lead to bugs is that they are created when the function is created, and persist between function calls. If you happen to modify the list of `columns`, for example, by adding an element to it, then the next time the function is called `columns` will include that element.

Reply:
I used something similar to the second method.
Thank you for your comment.
If you have time, check the new commit.